src/pages/projectSettings/tabs/EventLogTab/EventLogDiffs.test.ts

```diff
@@ -0,0 +1,203 @@
+      message: "",
+    },
+  },
+  subscriptions: [],
+  vars: {
+    __typename: "ProjectVars",
+    vars: { newVariable: "so new" },
+    privateVars: [],
+    adminOnlyVars: [],
+  },
+  aliases: [],
+  githubWebhooksEnabled: false,
+};
+const after = {
+  __typename: "ProjectEventSettings",
+  projectRef: {
+    __typename: "Project",
+    identifier: "viewTest",
+    repoRefId: "",
+    versionControlEnabled: false,
+    triggers: [],
+    tracksPushEvents: null,
+    hidden: false,
+    enabled: true,
+    owner: "a",
+    repo: "a",
+    branch: "main",
+    displayName: "",
+    batchTime: 0,
+    remotePath: "",
+    spawnHostScriptPath: "",
+    dispatchingDisabled: false,
+    deactivatePrevious: false,
+    repotrackerDisabled: false,
+    defaultLogger: "buildlogger",
+    validDefaultLoggers: ["evergreen", "buildlogger"],
+    cedarTestResultsEnabled: false,
+    patchingDisabled: false,
+    taskSync: {
+      __typename: "TaskSyncOptions",
+      configEnabled: null,
+      patchEnabled: null,
+    },
+    disabledStatsCache: false,
+    filesIgnoredFromCache: [],
+    private: false,
+    restricted: false,
+    admins: [],
+    buildBaronSettings: {
+      __typename: "BuildBaronSettings",
+      ticketCreateProject: "",
+      ticketSearchProjects: null,
+    },
+    taskAnnotationSettings: {
+      __typename: "TaskAnnotationSettings",
+      jiraCustomFields: null,
+      fileTicketWebhook: { __typename: "Webhook", endpoint: "", secret: "" },
+    },
+    perfEnabled: false,
+    notifyOnBuildFailure: false,
+    patchTriggerAliases: [
+      {
+        __typename: "PatchTriggerAlias",
+        alias: "noLongerNewAlias",
+        childProjectIdentifier: "evg",
+        taskSpecifiers: [
+          {
+            __typename: "TaskSpecifier",
+            patchAlias: "__github",
+            taskRegex: "",
+            variantRegex: "",
+          },
+        ],
+        status: "*",
+        parentAsModule: "",
+      },
+    ],
+    githubTriggerAliases: [],
+    workstationConfig: {
+      __typename: "WorkstationConfig",
+      gitClone: null,
+      setupCommands: null,
+    },
+    periodicBuilds: [],
+    prTestingEnabled: false,
+    githubChecksEnabled: false,
+    gitTagVersionsEnabled: false,
+    gitTagAuthorizedUsers: [],
+    gitTagAuthorizedTeams: [],
+    commitQueue: {
+      __typename: "CommitQueueParams",
+      enabled: null,
+      requireSigned: null,
+      mergeMethod: "squash",
+      message: "",
+    },
+  },
+  subscriptions: [],
+  vars: {
+    __typename: "ProjectVars",
+    vars: { newVariable: "so new" },
+    privateVars: [],
+    adminOnlyVars: [],
+  },
+  aliases: [],
+  githubWebhooksEnabled: false,
+};
+
+describe("event diff lines", () => {
+  it("should transform event diffs to key, before and after", () => {
+    const diffLines = getEventDiffLines(before, after);
+    expect(diffLines).toStrictEqual([
+      {
+        key: "projectRef.patchTriggerAliases[0]alias",
```

Comment:
There should still be a period after the `]` bracket since that's how you would access a property of an object in an array

Reply:
done